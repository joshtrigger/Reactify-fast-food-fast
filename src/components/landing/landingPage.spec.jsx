import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import LandingPage  from "./landingPage";

configure({ adapter: new Adapter() });

let wrapper;

describe("test ladning page", () => {
  beforeEach(() => {
    wrapper = shallow(<LandingPage />);
  });
  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
