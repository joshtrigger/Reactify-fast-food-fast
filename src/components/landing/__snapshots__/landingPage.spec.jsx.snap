// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ladning page should match snapshot 1`] = `ShallowWrapper {}`;
